lib/grpc-api-assistant/grpc_steps.rb

```diff
@@ -6,9 +6,9 @@
   @stored_known_values[name] = Helpers.get_env(name) || name + "_" + SecureRandom.uuid
 end
 
-Given /^the value '(.+)' is saved in a key called '(.+)'$/ do |value, name|
+Given /^the value '(.*)' is saved in a key called '(.+)'$/ do |value, name|
```

Comment:
Allow for blank or empty strings to avoid adding this step:
```
Given(/^the value '' is saved in a key called '(.+)'$/) do |name|
  @stored_known_values[name] = ""
end
```